.github/workflows/ci-async.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches: [ staging, trying, master ]
+  pull_request:
+
+name: Continuous integration
+
+jobs:
+  ci-linux-async:
```

Comment:
Can you please merge these changes into `ci.yml` as a separate job?

Reply:
Yup, sure. Was just copying this pattern from embedded-hal-async :D I'll merge them both